docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -11,6 +11,7 @@ aliases:
 # Getting started with vmanomaly
 
 **Prerequisites**
+- *vmanomaly* is a part of enterprise package. You can get license key [here](https://victoriametrics.com/products/enterprise/trial) to try this tutorial.
 - In the tutorial, we'll be using the following VictoriaMetrics components:
   -  [VictoriaMetrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html) (v.1.83.1)
```

Comment:
should we refresh our own components' versions?
i.e. to 1.93.1, which is [the freshest one according to our releases](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) (but omitting `latest` tag) 

here and throughout this guide